src/resources/js/layouts/sessions/main.vue

```diff
@@ -25,10 +25,10 @@
                                 />
                                 <small>
                                     {{
-                                        session.testCases
-                                            ? collect(session.testCases.data)
-                                                  .unique('use_case_id')
-                                                  .count()
+                                        session.testCases.data
+                                            ? collect(session.testCases.data).map(function ($value) {
+                                                return $value['useCase']['id'];
+                                            }).unique().count()
```

Comment:
Minor suggestion, but `$foo` is not a very javascript-y naming convention these days! Additionally where object keys are simple strings, it's normally better to just use dot notation (I think `prettier` might do this for you if you configure your IDE to format with prettier on save)

```suggestion
                                            ? collect(session.testCases.data).map(function (value) {
                                                return value.useCase.id;
                                            }).unique().count()
```